scripts/build-ocl-windows.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -o xtrace
+
+git clone --branch v2022.01.04 https://github.com/KhronosGroup/OpenCL-ICD-Loader
+
+git clone --branch v2022.01.04 https://github.com/KhronosGroup/OpenCL-Headers
+
+
+cmake -D CMAKE_INSTALL_PREFIX=./OpenCL-Headers/install -S ./OpenCL-Headers -B ./OpenCL-Headers/build 
+cmake --build ./OpenCL-Headers/build --target install
+
+# if someone would like to try to create win32 wheels bellow lines may be usefull
+# cmake -D CMAKE_PREFIX_PATH=${PWD}/OpenCL-Headers/install -DOPENCL_ICD_LOADER_HEADERS_DIR=${PWD}/OpenCL-Headers/install/include -S ./OpenCL-ICD-Loader -B ./OpenCL-ICD-Loader/build 
+# cmake --build ./OpenCL-ICD-Loader/build --target install --config Release
+
+cmake -D CMAKE_PREFIX_PATH=${PWD}/OpenCL-Headers/install -D OPENCL_ICD_LOADER_HEADERS_DIR=${PWD}/OpenCL-Headers/install/include -S ./OpenCL-ICD-Loader -B ./OpenCL-ICD-Loader/build2 -A x64
+cmake --build ./OpenCL-ICD-Loader/build2 --target install --config Release
```

Comment:
Are there license files that should be included here?

Reply:
will check tomorrow.